docs/api-reference/carto/data-sources.md

```diff
@@ -147,19 +144,50 @@ type quadbinQuerySourceOptions = {
 #### quadbinTilesetSource
 
 ```ts
-type quadbinTilesetSourceOptions = {
+type QuadbinTilesetSourceOptions = {
   tableName: string;
 }
 ```
 
 #### rasterTilesetSource
 
 ```ts
-type rasterTilesetSourceOptions = {
+type RasterTilesetSourceOptions = {
+  tableName: string;
+}
+```
+
+#### boundaryTableSource
+
+```ts
+type BoundaryTableSourceOptions = {
+  boundaryId: string;
+  columns?: string[];
+  matchingColumn?: string;
   tableName: string;
 }
 ```
 
+#### boundaryQuerySource
+
+```ts
+type BoundaryQuerySourceOptions = {
+  boundaryId: string;
+  matchingColumn?: string;
+  sqlQuery: string;
+  queryParameters: QueryParameters;
+}
+```
+
+#### boundaryTilesetSource
+
+```ts
+type BoundaryTilesetSourceOptions = {
+  boundaryId: string;
+  columns?: string[];
```

Comment:
Why is this required?  In tileset we don't support columns as the API cannot filter by them